### ISSUE TITLE
Dynamic content changes are not reported in Braille.

### ISSUE THREAD

**Issue by rperez030** [nvaccess/nvda#13800]
when the NVDA setting to report dynamic content changes is on, these changes should appear as flash messages in Braille. This can be seen as an enhancement, but I am reporting it as a bug because it can create a significant barrier for screen reader users that are deaf-blind or braille only users, particularly when dynamic changes are not persistent on the screen or when the new content is not nearby the element with focus. The enhancement in this case could be an optional setting to report dynamic content changes in Braille  
### Steps to reproduce:
1. Using NVDA with a Braille display, open CMD.
2. Verify that Report Dynamic content changes is on by pressing NVDA +5.
3. Type a command such as "ver".
### Actual behavior:
The output of the command will be spoken but not presented in Braille. Braille users will have to navigate the screen to find the new content. In situations where the content is not persistent this can constitute a blocker.
### Expected behavior:
The output of the command will appear in braille as a flash message.
#### NVDA installed/portable/running from source:
N/A
#### NVDA version:
2022.1
#### Windows version:
N/A
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
N/A
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
@rperez030 is this still reproducible in NVDA 2024 4 Beta?
cc: @burmancomp, @Emil-18, @codeofdusk could anyone of you test as well please?

**Comment by Emil-18** [comment]
@Adriani90 It is reproducible in the latest beta version of NVDA.
To fix this however, I think we need to change how braille.handler.message or the LiveText class works. If we just report the spoken content in braille with ui.message, it will only show the last spoken text in braille.
For example, in the following code, braille would only have shown "Text3" or ">>>"
print("Text1\nText2\nText3")

**Comment by v-rperez030** [comment]
> For example, in the following code, braille would only have shown "Text3" or ">>>" print("Text1\nText2\nText3")

Exactly. Ideally, the Braille gestures to move to previous and next line should explore the message until it is dismissed. I wonder if a solution could be to simply drop the line breaks from the text sent to the Braille display so that each call to ui.message produces one line of text only.
